### PR TITLE
Duplicate permission step added to permission test

### PR DESCRIPTION
**Pull request by Valentina1133**
### What does the PR do

Added steps to verify duplication of permission

CI run:
https://ci.status.im/job/status-desktop/job/e2e/job/manual/2170/

### CODE REVIEW

**Review by anastasiyaig — APPROVED** [review]

File: test/e2e/gui/objects_map/communities_names.py [hunk 1]
```diff
@@ -178,10 +178,12 @@
...
 hide_icon_StatusIcon = {"container": mainWindow_PermissionsView, "objectName": "hide-icon", "type": "StatusIcon", "visible": True}
+copy_icon_StatusIcon = {"container": mainWindow_PermissionsView, "objectName": "copy-icon", "type": "StatusIcon", "visible": True}
 editPermissionView_settingsDirtyToastMessage_SettingsDirtyToastMessage = {"container": mainWindow_editPermissionView_EditPermissionView, "id": "settingsDirtyToastMessage", "type": "SettingsDirtyToastMessage", "unnamed": 1, "visible": True}
 update_StatusButton = {"checkable": False, "container": statusDesktop_mainWindow_overlay, "type": "StatusButton", "unnamed": 1, "visible": True}
 isAllowedToEditPermissionView_StatusListItemTag = {"container": mainWindow_editPermissionView_EditPermissionView, "type": "StatusListItemTag", "unnamed": 1, "visible": True}
 memberRoleLimitWarning = {"container": mainWindow_editPermissionView_EditPermissionView, "id": "duplicationPanel", "type": "WarningPanel", "unnamed": 1, "visible": True}
+editPermissionView_duplicationPanel_WarningPanel = {"container": mainWindow_editPermissionView_EditPermissionView, "id": "duplicationPanel", "type": "WarningPanel", "unnamed": 1, "visible": True}
```
Comment (anastasiyaig) [inline test/e2e/gui/objects_map/communities_names.py:186]: maybe adding object name here?

Comment (Valentina1133) [reply test/e2e/gui/objects_map/communities_names.py]: Added

**Review by Seitseman — APPROVED** [review]

**Review by Seitseman — APPROVED** [review]

**Comment by anastasiyaig** [comment]
permission test is failing here

**Comment by Valentina1133** [comment]
> permission test is failing here

Yes, I see that test for 6th member role permission test failed (not the one I changed). Going to check what is wrong there

**Comment by Valentina1133** [comment]
> permission test is failing here

Ok. Fixed. There were 2 methods and 2 locators (1 - for limit warning, 1 for duplicate warning), but it's actually the same object. So I reworked it a bit so we use one for both warnings

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**